src/components/SearchBar.tsx

```diff
@@ -0,0 +1,33 @@
+import  { useState, ChangeEvent, FC } from 'react';
+
+interface SearchBarProps {
```

Comment:
Can we go with `SearchBarProp` ?
```suggestion
interface SearchBarProp {
```

Reply:
change https://github.com/Sakshi-Simform/Recipe-App/pull/3/commits/71e535163e542d10351e1face0f766e7a028837b